modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -174,7 +175,14 @@ public SW360Release setChangeStatus(String changeStatus) {
     }
 
     public Map<String, String> getExternalIds() {
-        return externalIds;
+        if(externalIds != null) {
```

Comment:
Alternatively (but possibly not better): 
```suggestion
        return Optional.of(externalIds)
            .map(Map::entrySet)
            .map(Collection::stream)
            .orElse(Stream.empty())
            .filter(stringStringEntry -> !stringStringEntry.getValue().equals("")
            .collect(Collectors.toMap(Map.Entry::getKey,
￼                             Map.Entry::getValue)
```

This solution returns an empty map, if external IDs is `null`, which might be better.